planner/core/rule_join_elimination.go

```diff
@@ -199,7 +199,10 @@ func (o *outerJoinEliminator) doOptimize(p LogicalPlan, aggCols []*expression.Co
 			parentCols = append(parentCols, expression.ExtractColumns(expr)...)
 		}
 	case *LogicalAggregation:
-		parentCols = append(parentCols[:0], x.groupByCols...)
```

Comment:
A question won't block merging this pr: could we just remove the `groupByCols` from the `Aggregation`?

Reply:
I think we can't. `groupByCols` are used to check whether aggregation can keep order when generating stream agg.